i18n/zh/docusaurus-plugin-content-docs/current/userguide/globalview/customizing-resource-interpreter.md

```diff
@@ -536,3 +536,18 @@ Kyverno的`ClusterPolicy`是一个规则的集合，它不依赖于其他资源
 [15]: https://github.com/karmada-io/karmada/blob/master/pkg/apis/config/v1alpha1/resourceinterpretercustomization_types.go#L79-L84
 [16]: https://github.com/karmada-io/karmada/blob/master/pkg/apis/config/v1alpha1/resourceinterpretercustomization_types.go#L94-L97
 [17]: https://github.com/karmada-io/karmada/blob/master/pkg/apis/config/v1alpha1/resourceinterpretercustomization_types.go#L99-L105
+
+## 注意事项
```

Comment:
English documents also need to be modified.

Reply:
Yes, I would prefer you reading the description of the Chinese version first, after confirming that there is no problem, I will continue to submit the English version of the PR.